### DISCUSSION TITLE
How to use the kni content builder with the mgcb-editor?

### DISCUSSION THREAD

**Discussion by aCCrab** [kniEngine/kni#1876] [Content Pipeline]
Hi, I wonder if I have missed an installation step to use the mgcb-editor, because I can't seem to make it work.

Here's what I have done:
1. Install the latest kni SDK
2. Created a new BlazorGL project with VS2022
3. Used `mgcb-editor-windows.exe \Content\Content.mgcb`
But a pop up tells me :
>  Invalid value 'BlazorGL' for option 'platform'

I guess it's because it's only the monogame content builder and not the kni content builder. Thus, I tried building content directly with `'C:\Program Files (x86)\MSBuild\MonoGame\v3.0\Tools\MGCB.exe' /build:<resource>` which seems to work but also seems overly complicated and I would prefer using the GUI editor. 

So, is it possible to use the mgcb-editor with the kni content builder, and if so, could you tell me how please?
Thank you!

**Comment by nkast** [comment]
The KNI sdk comes with it's own PipelineEditor.
Press 'Start' and type 'KNI PipelineEditor'.
Normally it should also register it as the default app for .mgcb, Is it not?

If you want to also edit the .mgcb with mgcb-editor you can simply edit 
the .mgcb file and remove the line '/Platform:BlazorGL'. 
Actually the platform is specified from the .csproj during a build, it's not really required to have it inside the .mgcb.
